net/unbound/files/unbound.sh

```diff
@@ -1067,28 +1065,33 @@ unbound_start() {
 
     unbound_control
   fi
+
+  if unbound_ismain ; then
+    # set resolver file to local.
+    rm -f /tmp/resolv.conf
+
+    {
+      echo "nameserver 127.0.0.1"
+      echo "nameserver ::1"
+      echo "search $UNBOUND_TXT_DOMAIN"
+    } > /tmp/resolv.conf
+  fi
 }
 
 ##############################################################################
 
 unbound_stop() {
-  local resolvsym=0
-
-  rootzone_update
-
-
-  if [ ! -x /usr/sbin/dnsmasq -o ! -x /etc/init.d/dnsmasq ] ; then
-    resolvsym=1
-  else
-    /etc/init.d/dnsmasq enabled || resolvsym=1
-  fi
+  config_load unbound
+  config_foreach unbound_uci unbound
```

Comment:
I had a few minutes and was trying to provide some quick feedback and hints from my smart phone. So now based on previous comment for _resolv_setup() lets create _resolv_teardown(). It doesn't use any UCI. It just sees if Unbound did the resolve.conf update.
```
case $( cat /tmp/resolve.conf ) in 
*"generated by Unbound UCI"*)
   rm -f /tmp/resolv.conf
   ln -s /tmp/resolv.conf.auto /tmp/resolv.conf
  ;;
esac
```